modules/account-lib/src/coin/sol/ataInitializationBuilder.ts

```diff
@@ -10,13 +10,15 @@ import { BuildTransactionError } from '../baseCoin/errors';
 import { ASSOCIATED_TOKEN_PROGRAM_ID, Token, TOKEN_PROGRAM_ID } from '@solana/spl-token';
 import assert from 'assert';
 import { SolCoin } from '@bitgo/statics/dist/src/account';
+import { AtaInitializationTransaction } from './ataInitializationTransaction';
 
 export class AtaInitializationBuilder extends TransactionBuilder {
   private _mint: string;
   private _rentExemptAmount: string;
 
   constructor(_coinConfig: Readonly<CoinConfig>) {
     super(_coinConfig);
+    this._transaction = new AtaInitializationTransaction(_coinConfig);
```

Comment:
is it possible to re-declare the `_transaction` property as `AtaInitializationTransaction`?
e.g.

`private _transaction: AtaInitializationTransaction`